internal/internal_worker_base.go

```diff
@@ -27,7 +27,9 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/shirou/gopsutil/cpu"
```

Comment:
Standart imports should be separated from external. I think goimports should do the trick